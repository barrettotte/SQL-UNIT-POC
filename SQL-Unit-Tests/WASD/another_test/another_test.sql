-- Generated with SQL-Unit --
SELECT * FROM [BARRETT_TEST].[dbo].[users];