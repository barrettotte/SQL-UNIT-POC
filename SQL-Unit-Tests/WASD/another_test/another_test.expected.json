{
    "columns": [
        {
            "name": "role_name",
            "type": "VARCHAR(50)"
        }
    ],
    "row-count": 10,
    "success": "true",
    "result-set": [
        {
            "id": 1,
            "role_name": "ROLE_USER"
        },
        {
            "id": 2,
            "role_name": "ROLE_ADMIN"
        },
        {
            "id": 3,
            "role_name": "ROLE_DEVELOPER"
        }
    ]
}